angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -170,6 +170,10 @@
                 <nimble-list-option *ngFor="let item of comboboxItems" [ngValue]="item">{{ item ? item.first : '' }}</nimble-list-option>
             </nimble-combobox>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Rich Text Viewer</div>
+            <nimble-rich-text-viewer markdownValue='Markdown value'></nimble-rich-text-viewer>
```

Comment:
I understand that we are yet to implement `markdownValue` property inputs inside `nimble-components` and `nimble-angular`. With this, do we still need to send an non-existing input `markdownValue` already in this PR, which will be anyway ignored for rendering? 

Let me know if we will update this PR after #1335.

After the `markdownValue` property setup, consider sending a more representative markdown input similar to `I am a rich text viewer. I can help you format text in **Bold**, _Italics_,...` as an input.

Reply:
I have removed the `markdownValue` property from the example-client-app. 

Yes, we will update it once https://github.com/ni/nimble/pull/1335 is complete.